app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -162,6 +163,22 @@ public boolean onActionItemClicked(ActionMode inputMode, MenuItem inputItem) {
 
                 return true;
             }
+            else if(inputItem.getItemId() == R.id.action_archive_unarchive){
+                for (LoyaltyCard loyaltyCard : mAdapter.getSelectedItems()) {
+                    Log.e(TAG, "Archiving card: " + loyaltyCard.id);
```

Comment:
e in `Log.e` means error. It's not an error. `Log.d` would make more sense

Reply:
I see I messed that up for card deletion too, but better to fix that too than to make it wrong twice :)